botocore/compat.py

```diff
@@ -25,19 +26,17 @@
 from collections.abc import MutableMapping
 from math import floor
 
-from botocore.vendored import six
 from botocore.exceptions import MD5UnavailableError
 from dateutil.tz import tzlocal
 from urllib3 import exceptions
 
 logger = logging.getLogger(__name__)
 
 
-from botocore.vendored.six.moves import http_client
-
-class HTTPHeaders(http_client.HTTPMessage):
+class HTTPHeaders(http.client.HTTPMessage):
     pass
 
+
```

Comment:
There's a number of linting changes I wouldn't expect in this PR. Could you review the [CONTRIBUTING](https://github.com/boto/botocore/blob/develop/CONTRIBUTING.rst#codestyle) guide for how to format your changes and let us know if you have any questions. Thanks!

Reply:
All linting changes were done only to make your CI happy. I don’t understand. As far as I can see your CI is happy.

```
stitny~/r/t/botocore (no-six)$ pre-commit run -a
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
isort....................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

botocore/paginate.py:252:5: C901 'PageIterator.__iter__' is too complex (11)
tests/unit/test_protocols.py:181:1: C901 'test_output_compliance' is too complex (11)
botocore/configloader.py:203:1: C901 'build_profile_map' is too complex (11)
botocore/httpsession.py:439:5: C901 'URLLib3Session.send' is too complex (11)
botocore/args.py:157:5: C901 'ClientArgsCreator.compute_client_args' is too complex (12)
botocore/utils.py:1159:5: C901 'ArgumentGenerator._generate_skeleton' is too complex (12)

pyupgrade................................................................Passed
black....................................................................Passed
stitny~/r/t/botocore (no-six)$ 
```

Reply:
Apologies, I should have expounded more. There are formatting changes to compat.py, as well as the `vendored` and `docs` directories. These are excluded in the [`.pre-commit-config.yml`](https://github.com/boto/botocore/blob/develop/.pre-commit-config.yaml#L7-L8) which suggested black was run across the whole project rather than pre-commit.

Reply:
If there is no interest in this PR at all, then I don’t think it matters what indentation it has. And yes, I run black on whole project once (I guess `git checkout develop -- botocore/vendored/*` would help to restore).